authors/urls.py

```diff
@@ -34,11 +34,9 @@
     path('admin/', admin.site.urls),
 
     path('api/', include(('authors.apps.authentication.urls',
-                          'authentication'), namespace='authentication')),
-    path('apidocs/', schema_view.with_ui('swagger',
-                                         cache_timeout=0),
-         name='schema-swagger-ui'),
-    path('docs/', include_docs_urls(
-        title="AQUAMAN AUTHOR'S HAVEN API",
-        description="A place where different authors pen down their thoughts"))
+                           'authentication'), namespace='authentication')),
+    path('apidocs/',schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+    path('docs/', include_docs_urls(title="AQUAMAN AUTHOR'S HAVEN API",
+                    description="A place where different authors pen down their thoughts")),
+    path('api/social/', include('authors.apps.social_auth.urls')),
```

Comment:
So here I was thinking maybe you could add a namespace this can be very helpful when you don't want to write the full route instead you would just call that namespace this is just for your consideration